### ISSUE TITLE
Support for more language codes

### ISSUE THREAD

**Issue by beer-psi** [Aidoku/Aidoku#62]
Currently, Aidoku only supports a [select number of language codes](https://github.com/Aidoku/Aidoku/blob/main/Shared/Sources/SourceManager.swift#L20-L22). This can be problematic if a source with a language outside the list shows up.

**Comment by Skittyblock** [comment]
Yes, the available language codes should be fetched from the added source list(s).

**Comment by Skittyblock** [comment]
this should be fixed now; language codes should load from added source lists with 0.7.
